app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,18 @@
+class GroupsController < ApplicationController
+  def index
+    @groups = Group.includes(:entities).where(author: current_user)
+  end
+
+  def new; end
+
+  def create
+    current_user.groups.create(group_params)
+    redirect_to groups_path
+  end
+
```

Comment:
- [ ] It is always a great practice to create success and failure paths. What if the group is not created for some reason? You should also handle that scenario.